Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM fedora:latest
+MAINTAINER Red Hat, Inc. <container-tools@redhat.com>
+
+ENV MHM_RELEASE v0.1.0-alpha
+ENV PYTHONPATH  /commissaire-http/src/
+
+# Install required dependencies and commissaire-http
```

Comment:
Could you add a comment here about why `gcc`, `redhat-rpm-config` and `libffi-devel` are temporarily needed?  I'm not sure of the reason myself.  `git` is obviously needed for `git clone`.  Are the others just dependencies?